tools/publish_aws.py

```diff
@@ -42,7 +42,7 @@ def __init__(
         self._universe_url_prefix = os.environ.get(
             'UNIVERSE_URL_PREFIX',
             'https://universe-converter.mesosphere.com/transform?url=')
-        s3_bucket = os.environ.get('S3_BUCKET', 'infinity-artifacts')
+        s3_bucket = os.environ.get('S3_BUCKET') if os.environ.get('S3_BUCKET') else 'infinity-artifacts'
```

Comment:
I would almost consider doing this in two steps. The line gets a little long otherwise:
```python
s3_bucket = os.environ.get('S3_BUCKET')
if not s3_bucket:
    s3_bucket = 'infinity-artifacts'
log.info('Using artefact bucket: %s', s3_bucket)
```

Reply:
In general I try to avoid multiple assignments to the same variable but this is simple enough: https://github.com/mesosphere/dcos-commons/pull/2038/commits/ac0372f42bbde69c54f0969be151fb7e2633497b